src/sentry/api/endpoints/codeowners/details.py

```diff
@@ -11,19 +14,25 @@
 from sentry.api.exceptions import ResourceDoesNotExist
 from sentry.api.serializers import serialize
 from sentry.api.serializers.models import projectcodeowners as projectcodeowners_serializers
-from sentry.models import ProjectCodeOwners
+from sentry.models import Project, ProjectCodeOwners
 
 from . import ProjectCodeOwnerSerializer, ProjectCodeOwnersMixin
 
 logger = logging.getLogger(__name__)
 
 
 class ProjectCodeOwnersDetailsEndpoint(
-    ProjectEndpoint, ProjectOwnershipMixin, ProjectCodeOwnersMixin
+    ProjectEndpoint, ProjectOwnershipMixin, ProjectCodeOwnersMixin  # type: ignore
 ):
     def convert_args(
-        self, request: Request, organization_slug, project_slug, codeowners_id, *args, **kwargs
-    ):
+        self,
+        request: Request,
+        organization_slug: str,
+        project_slug: str,
+        codeowners_id: int,
```

Comment:
Is this actually an int? Afaik, variable extracted from urls are strings.

Reply:
👍🏽  you're right. I was looking at it from the model side.